.github/actions/jx-updatebot-pr/action.yml

```diff
@@ -78,6 +78,14 @@ inputs:
     required: false
     description: project root directory. Defaults to '.'
     default: '.'
+  retries:
+    description: 'How many times to retry on failure'
```

Comment:
With the term "retry", I feel like as a user, I would need to put "0" to avoid retrying (eg run only once), but given the implementation, it would need to be 1, correct?
If so, maybe the implementation should be changed to add + 1 to this input.
Also, I'm in favor of not retrying by default.

Reply:
(but if the default is something we need to manage globally anyway, I'm ok keeping 5, it really depends on the reasons behind the need to retry here: can you elaborate on the problem please?)